src/Tests/SettingsTest.php

```diff
@@ -29,7 +29,7 @@ protected function setUp() {
 
     $permissions = [
       'access administration pages',
-      'administer acquia lift',
+      'administer acquia_lift',
```

Comment:
Good change.

I think we can still make this change without update hook, because chances are slight that a customer site has a non-admin user that was explicitly assigned 'administer acquia lift'. At least I haven't seem any.